src/contrib/parallel-rdp/parallel-rdp-standalone/COMMIT

```diff
@@ -1 +1 @@
-fe5becd13638873db90d46e7ba7d48255971f82a
+1f69c762be68feb9fcd5276d75acc6e5a6160a19
```

Comment:
Swapping out the subtree for my own fork, incidentally, made the subtree more up to date